### PR TITLE
Manifest::from_remote integration test

### PR DESCRIPTION
**Pull request by tarrencev**
updates blockifier + updates world constructor to not take name + implements migration integration test

### CODE REVIEW

File: crates/dojo-world/src/migration/strategy.rs [hunk 1]
```diff
@@ -67,11 +67,6 @@ impl MigrationStrategy {
                     res.declare_res.transaction_hash, res.contract_address
                 );
 
-                if self.world.is_none() {
-                    let addr = self.world_address().ok_or(MigrationError::WorldAddressNotFound)?;
-                    WorldContract::new(addr, &migrator).set_executor(res.contract_address).await?;
-                }
```
Comment (tarrencev) [inline crates/dojo-world/src/migration/strategy.rs:73]: read code? @kariy

Comment (kariy) [reply crates/dojo-world/src/migration/strategy.rs:73]: its for when the `executor` need to be migrated but not the `world`.

Comment (kariy) [reply crates/dojo-world/src/migration/strategy.rs:73]: this is for when there's already exist a remote world

Comment (tarrencev) [reply crates/dojo-world/src/migration/strategy.rs:73]: hm but does it actually do something? is there some sort of side effect?

Comment (kariy) [reply crates/dojo-world/src/migration/strategy.rs:73]: its supposed to set the new executor to the existing world. is that not meant to happen?

Comment (tarrencev) [reply crates/dojo-world/src/migration/strategy.rs:73]: ah word. my bad, i thought it was just setting the value on a rust struct. didn't realize it was making a txn

**Pull request closed without merging**